cmd/healthbot/main.go

```diff
@@ -49,7 +49,8 @@ func main() {
 			log.Fatal().Int("chain_id", chainCfg.ChainID).Msg("the chain id isn't supported in the Tableland client")
 		}
 
-		client, err := clientV1.NewClient(ctx, wallet, clientV1.NewClientChain(chain))
+		client, err := clientV1.NewClient(
+			ctx, wallet, clientV1.NewClientChain(chain), clientV1.NewClientAlchemyAPIKey(chainCfg.AlchemyAPIKey))
```

Comment:
This is how I noticed the missing `AlchemyAPIKey` above. 

I think we can improve the UX in this client API by thinking of not making the API Key being an optional.
Despite being an optional is nice to have different flavors of "Infura", "Alchemy", etc, it's isn't correct in the sense that it isn't "optional". After you run it, it will break since it can't connect to any Ethereum API provider.

Reply:
Created https://github.com/tablelandnetwork/go-tableland/issues/441 to discuss.